### ISSUE TITLE
[Legitimate Site Blocked] www.metaosk.com

### ISSUE THREAD

**Issue by Hickshu** [MetaMask/eth-phishing-detect#7223]
Devenblake Hello.
Metaosk.com is a healthy and legally certified official website, please help us so that our domain can be unblocked.

Our official website is legal and has SSL certification, but now whenever you start Metaosk.com from your computer, you will be warned that it is a phishing website, which causes many users to distrust us, so we hope that our domain can be officially whitelisted.

**Comment by devenblake** [comment]
Duplicate of #7122.
